torch/jit/frontend.py

```diff
@@ -266,6 +266,15 @@ def build_Return(ctx, stmt):
         values = (stmt.value,) if not isinstance(stmt.value, ast.Tuple) else stmt.value.elts
         return Return(r, [build_expr(ctx, val) for val in values if val is not None])
 
+    @staticmethod
+    def build_Raise(ctx, stmt):
+        r = ctx.make_range(stmt.lineno, stmt.col_offset, stmt.col_offset + len("raise"))
+        if isinstance(stmt.exc, ast.Name):
```

Comment:
It is weird to check for ast.Name but not to disallow all the other expressions that we do not handle (`1`, `a + b`, etc.). For now, we allow anything at this stage, and then just ignore the expression.